knx/dpt/formats.go

```diff
@@ -4,11 +4,13 @@
 package dpt
 
 import (
+	"encoding/binary"
 	"errors"
+	"math"
 )
 
 // ErrInvalidLength is returned when the application data has unexpected length.
-var ErrInvalidLength = errors.New("Given application data has invalid length")
+var ErrInvalidLength = errors.New("given application data has invalid length")
```

Comment:
Why?

Reply:
My IDE considers this as a code smell issue regarding the error convention, i can revert it if you prefer?

Reply:
Keep it, it is fine. I was just not expecting it given it is completely unrelated to the change.